src/commands/add/addCredential.ts

```diff
@@ -17,8 +17,13 @@ import { getAndValidateServerUrl, getCredentialsInput } from './internal/input'
  * The file will contain the client ID, secret, access token and refresh token.
  * Its name will be of the form `.env.{targetName}`
  * @param {string} targetName- the name of the target to create the env file for.
+ * @param {boolean} insecure- boolean to use insecure connection, default is false.
```

Comment:
As a reader I would be curious what an insecure connection means, and why we would need it. Can we provide a link to further information?  Eg the adapter code, or documentation, or the updated page on cli.sasjs.io ?

Reply:
or, (or in addition) could of course provide more information here.

Reply:
Updating to: 
* @param {boolean} insecure- boolean to use insecure connection, default is false. If true the cli will not reject server connection which is not authorized with the list of supplied CAs.